resolwe/toolkit/tools/parse_tabular_file.py

```diff
@@ -38,11 +36,10 @@ def main():
                         outline = line.strip().split(',')
                         csvwriter.writerow(outline)
             elif ext in ('.xls', '.xlsx'):
-                with open(args.input_file) as infile:
-                    workbook = xlrd.open_workbook(args.input_file)
-                    worksheet = workbook.sheets()[0]
-                    for rownum in range(worksheet.nrows):
-                        csvwriter.writerow(worksheet.row_values(rownum))
+                workbook = xlrd.open_workbook(args.input_file)
```

Comment:
@jvrakor, you are right, `xlrd.open_workbook()` doesn't hold the file open.

Here is a simple script that proves this:
```python
import psutil
import xlrd

proc = psutil.Process()

print(proc.open_files())

wb = xlrd.open_workbook('file tab.1.xlsx')

print(proc.open_files())

f = open('file tab.1.xlsx')

print(proc.open_files())
```

Its output is:
```
[]
[]
[popenfile(path='/home/tadej/Genialis/resolwe/resolwe/toolkit/tests/files/file tab.1.xlsx', fd=3, position=0, mode='r', flags=557056)]
```
which means that only the builin `open()` actually holds the file descriptor open.